libnavui-maps/src/main/res/drawable/mapbox_ic_camera_follow.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="32dp"
+    android:height="32dp"
+    android:viewportWidth="32"
+    android:viewportHeight="32">
+  <path
+      android:pathData="M15.3334,8.0001C14.9652,8.0001 14.6667,7.6684 14.6667,7.2593V1.4075C14.6667,0.9984 14.9652,0.6667 15.3334,0.6667H16.6667C17.0349,0.6667 17.3334,0.9984 17.3334,1.4075V7.2593C17.3334,7.6684 17.0349,8.0001 16.6667,8.0001H15.3334Z"
+      android:fillColor="#3C3E3E"
```

Comment:
Can we use the colors from colors.xml? Let's not worry about the colors in figma design because they are gonna come through NSS. Once available, we will change the colors for all views at once.

I would suggest to add a new property to `libnavui-maps/res/values/colors.xml`

```
<resources>
    <color name="mapbox_camera_mode_drawable_color">@color/colorOnSurface</color>
</resources>
```

Reply:
Icon colors are applied via `cameraModeButtonIconTint` attribute. I've updated `MapboxStyleCameraModeButton` style to apply `@color/colorOnSurface` tint by default.